ocaml/xenopsd/xc/service.ml

```diff
@@ -826,36 +870,15 @@ module PV_Vnc = struct
 
   let pid ~xs domid = D.pid ~xs domid
 
-  (* Look up the commandline args for the vncterm pid; *)
-  (* Check that they include the vncterm binary path and the xenstore console
-     path for the supplied domid. *)
-  let is_cmdline_valid domid pid =
```

Comment:
Thanks for spotting that we can reuse/repurpose this existing check.

Reply:
I found it when I was checking all the code which get process id from the function `pid` in base module `DaemonMgmt`, and luckily it can be reused. :)